src/components/study-container.js

```diff
@@ -199,24 +199,29 @@ export function StudyContainer({ view, onChangeTab }) {
         (isUpdate) => {
             console.info(`Loading network of study '${studyUuid}'...`);
 
-            if (isUpdate) {
-                // After a load flow, network has to be recreated.
-                // In order to avoid glitches during sld and map rendering,
-                // lines and substations have to be prefetched and set before network creation event is dispatched
-                // Network creation event is dispatched directly in the network constructor
-                new Network(
-                    studyUuid,
-                    workingNode?.id,
-                    (error) => {
-                        console.error(error.message);
-                        setNetworkLoadingFailMessage(error.message);
-                        //setIsNetworkPending(false);
-                    },
-                    dispatch,
-                    { equipments: [equipments.lines, equipments.substations] }
-                );
-            } else {
-                if (workingNode !== null) {
+            if (workingNode && studyUuid) {
```

Comment:
I would prefer something like if (!workingNode || !studyUuid) return;
as we don't do anything and add a level to indentation make the code less clear

Reply:
Modified.